en/frames_plane/turbo_timber_evolution.md

```diff
@@ -0,0 +1,203 @@
+- [Pixhawk 4 Mini/GPS/Power module/ set](https://shop.holybro.com/pixhawk4-mini_p1120.html)
+- SIK telemetry radio
+- MS4525DO differential pressure module and pitot tube
+- [Caddx Vista FPV air unit](https://caddxfpv.com/products/caddx-vista-kit)
+- [DJI FPV Goggles](https://www.dji.com/fpv)
+- [ExpressLRS Matek Diversity RX](http://www.mateksys.com/?portfolio=elrs-r24)
+- [Custom designed 3D printed parts](https://github.com/PX4/PX4-user_guide/raw/main/assets/airframes/fw/turbo_timber_evolution/3d_printed_parts.zip)
+  - Pixhawk 4 Mini mount and top GPS mount
+  - FPV pod and camera mount
+  - Pitot static pod and wing hardpoint hanger
+- Misc hardware: M3 hardware (standoffs, washers, bolts), XT30 connector, hot glue, heatshrink, Molex Microfit connectors
+- Silicone wiring (14awg for high current, 16awg for low current, 22awg for low power and signals)
+- 3.6Ah 4S LiPo OR 4s2p 18650 LiIon
+
+## Airframe Build
+
+The TTE comes out of the box very complete as-is.
+Servos and linkages have already been installed, and the only real work remaining is finishing work, such as installing the landing gear and horizontal stabilizer.
+For this portion of the assembly, I simply followed the manual.
+
+## FPV Pod
+
+![Window and front fuselage (hatch) with FPV Pod mounted on top](../../assets/airframes/fw/turbo_timber_evolution/fpv_pod_hatch.jpg)
+
+![Underside of hatch showing the FPV pod attachement screws and wires pulled through](../../assets/airframes/fw/turbo_timber_evolution/hatch_underside.jpg)
+
+The FPV pod was mounted on top of the battery hatch using M3 nylon hardware.
+Mounting holes for the pod were located by placing the FPV pod on top (being careful to center it with a ruler) and then punching through the fpv pod mounting holes into the foam with a screwdriver.
+A long M3 nylon screw and a washer on the underside, followed by a washer and standoff on the top of the battery hatch were then be used to mount the FPV pod.
+
+## Pitot Pod
+
+A airspeed sensor is highly recommended for use on fixed wing vehicles.
+This build uses a MS4525DO differential pressure module and pitot tube housed in a 3D printed pod that has a hardpoint hanger for connecting it to the wing.
+
+![Pitot pod/tube sitting on a desk](../../assets/airframes/fw/turbo_timber_evolution/pitotpod1.jpg)
+
+
+Inside the pitot pod, the MS4525DO differential pressure sensor is housed and is connected to the pitot/static tube with a short length of tubing.
+Zipties are used as hoseclamps to prevent the tubing from backing off.
+The i2c and power leads were soldered directly to the MS4525 module, and then hotglue was used to mechanically reinforce the connections.
+
+![Pitot pod opened up showing wires pitot tube connectors and wires soldered and hot-glued in place](../../assets/airframes/fw/turbo_timber_evolution/pitotpod2.jpg)

```

Comment:
Note, here I have split the description and the image - I think that makes it much easier to work out what the image is showing you, and why you provided it.

I am not proposing you do this, but what is even better than this is instructional steps like this:

1. Insert the wiring etc
    _image here_
2. Connect on wing to right of XYZ
    _image here_
3. And so on.

The reason is that if you have steps people can follow, they can't miss anything and neither can you. It makes the whole thing very structured and turnkey.

However as long as the information is here and the steps that need to be taken can be inferred, this is still able to pass the test "can people duplicate the work"